src/component/video.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+class Video extends React.Component{
+   render(){
+
+      if(!this.props.videoData){
+         return(
+            <div></div>
+         );
+      }
+      const {id,snippet} = this.props.videoData;
+
+      const videoSrc = `https://www.youtube.com/embed/${id}?autoplay=1`
```

Comment:
@RajanTank Same as my prev. comment. Check the same everywhere, because your code would break if the inputs are invalid.